bokeh/enums.py

```diff
@@ -7,9 +7,20 @@
 from . import colors, icons, palettes
 
 class Enumeration(object):
-    pass
 
-def enumeration(*values):
+    __slots__ = []
```

Comment:
Just preference. Since changing `__slots__` (if you do it) has no effect, I prefer a type that can't be changed.